platform/platform-scheduler/src/main/java/ddf/platform/scheduler/ScheduledTask.java

```diff
@@ -28,21 +28,21 @@
     /**
      * Creates and schedules new task
      */
-    public void newTask();
+    void newTask();
 
     /**
      * Removes a task completely so that it does not run or exist.
      *
      * @param code - not used
      * @see https://issues.apache.org/jira/browse/ARIES-1436
      */
-    public void deleteTask(int code);
+    void deleteTask(int code);
 
     /**
      * Updates an existing task with new properties.
      *
      * @param properties
      */
-    public void updateTask(Map properties);
+    void updateTask(Map<String, Object> properties);
```

Comment:
This packages does not appear to be in the whitelist for the Platform app.  I think this is a safe change to make but it would be good for someone to double check.  This is a good example of where using a consistent package structure for internal interfaces would have been useful.